mscore/scoreitemmodel.h

```diff
@@ -0,0 +1,53 @@
+//=============================================================================
+//  ScoreItemModel
+//
+//  Copyright (C) 2020 Peter Jonas
+//
+//  This program is free software; you can redistribute it and/or modify
+//  it under the terms of the GNU General Public License version 2
+//  as published by the Free Software Foundation and appearing in
+//  the file LICENCE.GPL
+//=============================================================================
+
+#ifndef __SCOREITEMMODEL_H__
+#define __SCOREITEMMODEL_H__
+
+#include "scoreview.h"
+#include <QAbstractItemModel>
```

Comment:
Should `<QAbstractItemModel>` be included here or added to [all.h](https://github.com/musescore/MuseScore/blob/4013ef92430373be44c9bc1156f81b58aaf7ba38/all.h#L120)? (CC. @anatoly-os)